charts/seed-monitoring/charts/core/charts/prometheus/templates/prometheus.yaml

```diff
@@ -89,8 +89,8 @@ spec:
             path: /-/healthy
             port: {{ .Values.port }}
             scheme: HTTP
-          failureThreshold: 60
-          periodSeconds: 5
+          failureThreshold: 3
+          periodSeconds: 100
```

Comment:
Please, revert this change. Now, I think it is better to tackle it via separate PR.